### ISSUE TITLE
Silent / Manual-mute can't be disabled

### ISSUE THREAD

**Issue by kopierschnitte** [MiRo1310/ioBroker.midas-aquatemp#24]
Hi!

For some reason, the silent mode (aka Manual-mute) can't be disabled. Enabling works without problems:

midas-aquatemp.0 | 2024-09-22 08:58:39.129 | info | DeviceStatus: {"sessionid":null,"error_code":"0","error_msg":"Success","error_msg_code":"","totalSize":null,"totalPage":null,"nextPage":null,"objectResult":null,"isReusltSuc":true}
-- | -- | -- | --
midas-aquatemp.0 | 2024-09-22 08:58:37.744 | debug | Silent: {"val":true,"ack":false,"ts":1726988317735,"q":0,"from":"system.adapter.web.0","user":"system.user.admin","lc":1726988317735}

But after setting it to false, the change won't trigger any actions and is therefore overwritten by true on the next cyclic query.

It looks to me like the adapter doesn't catch the user's change.

Tested with current 1.1.1 release.

**Comment by MiRo1310** [comment]
Kommt bei dir auch eine Fehlermeldung?

**Comment by kopierschnitte** [comment]
Nee, komischerweise gar nicht. Ich seh im Debug nur, dass beim nächsten Abfragen vom Server gemeldet wird, Manual-mute wäre noch 1.
Andersherum klappt es wunderbar. 

Könnte es sein, dass der Adapter gar nicht "UpdateDeviceSilent" sondern nur "UpdateDevicePower" aufruft?

**Comment by MiRo1310** [comment]
Ne der ruft das schon auf, beim mir bekomme ich aber einen Fehler beim axios request, das kann aber sein weil meine WP nicht mehr an ist. Ich werde das noch mal testen die Tage, ein bisschen was habe ich am Code auch noch angepasst. Das Problem ist das man keinen guten error response bekommt, wo man sehen kann was nicht funktioniert

**Comment by MiRo1310** [comment]
Ich habe mir gerade mal die chinesischen Zeichen übersetzen lassen. Das sagt nur man solle sich neu einloggen. Besser wäre wenn man das lokal ohne cloud nutzen kann. Kann man das denn nicht direkt mit dem wifi Modul ?

**Comment by kopierschnitte** [comment]
...oder, wir warten bis zur nächsten Saison. Der Rest funktioniert ja super!

**Comment by kopierschnitte** [comment]
Eine direkte Verbindung ist leider, wie so oft, völlig ausgeschlossen. Das Teil macht ja MQTT via SSL direkt nach China. Ärgerlich, aber wohl nicht zu ändern. Zumindest nicht ohne Zusatzhardware.

**Comment by kopierschnitte** [comment]
Im Prinzip einen Modbus/RS485-Adapter. Hab dazu was gefunden und im Forum hochgeladen:
https://forum.iobroker.net/post/1196308

**Comment by MiRo1310** [comment]
Ich hoffe das ich ab übernächster Woche hier weiter machen kann. Gibt es bei dir was neues? Evtl. doch alles ohne Cloud machen

**Comment by MiRo1310** [comment]
Werde ich dann ja sehen.

**Comment by MiRo1310** [comment]
@kopierschnitte Moin, wie schaut es bei dir jetzt aus, läuft das mit dem silent?

**Comment by MiRo1310** [comment]
@kopierschnitte Oh hatte garnicht mit bekommen das du geantwortet hattest, Gibt es eigentlich eine Doku zu der Api? Seit heute kann ich nämlich nicht mehr die Temperatur aus dem Adapter raus einstellen. Alles andere scheint auch nicht mehr zu gehen.In der App kann ich das ganze schon noch machen.

**Comment by kopierschnitte** [comment]
Gibt leider absolut gar keine offiziellen Informationen. Habe mich aber an dem Homeassistant-Projekt https://github.com/radical-squared/aquatemp orientiert. Die haben da wirklich eine Menge Energie hereingesteckt und bisher alle spontanen Änderungen eingepflegt.

Komischerweise habe ich derzeit gar keine Probleme. Läuft alles bestens und gibt auch keine Fehler mehr im Log. Temperatur und Silent-Mode kann ich prima einstellen und wird auch zeitnah abgefragt.

Vielleicht ist es nur ein vorübergehendes Problem?

Bei mir steigt es zum Beispiel völlig aus, wenn meine Internetleitung hakt und ich eine LTE-Ersatzverbindung benutze. Dann klappt jedoch auch die App nicht mehr.

**Comment by kopierschnitte** [comment]
Okay, jetzt hat es mich auch erwischt. Mist. Ich denke aber, die beim HA-Projekt werden das bald merken.

**Comment by MiRo1310** [comment]
@kopierschnitte Ja müssen wir dann mal beobachten
